system/jlib/jdebug.cpp

```diff
@@ -1650,8 +1651,18 @@ class CExtendedStats  // Disk network and cpu stats
 
     StringBuffer ifname;
 
+#ifdef __linux__
+    UnsignedArray majmin;
```

Comment:
should refer to 'disk' or 'storage' or similar in this var. name.

Reply:
Done